src/scripts/main.js

```diff
@@ -22,7 +22,20 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  element.append(list);
+
+  Object.entries(data).forEach(([key, value]) => {
+    const item = document.createElement('li');
+
+    item.textContent = key;
+    list.append(item);
+
+    if (Object.keys(value).length !== 0) {
```

Comment:
Replace with `Object.keys(value).length`, it will be the same as `Object.keys(value).length !== 0`